kolena/_experimental/object_detection/workflow.py

```diff
@@ -122,13 +120,9 @@ class TestCaseMetricsSingleClass(MetricsTestCase):
 
 @dataclass(frozen=True)
 class TestSampleMetrics(MetricsTestSample):
-    TP_labels: List[str]
```

Comment:
Do we lose visualization capability with this change? I suppose this is a quick way to see all labels

Reply:
You would have to visually scan the list of boxes, yeah. I think this is an acceptable tradeoff for reduced noise. If we wanted to preserve this functionality, we could extend the "X Bounding Boxes" summary to include the unique set of labels — this may be worth doing anyway to increase the signal in this collapsible header.